specification/riskiq/Easm/tspconfig.yaml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  "service-dir":
+    default: "sdk/easm"
+emit:
+  - "@azure-tools/typespec-apiview"
```

Comment:
```suggestion
```

Remove this emitter

Reply:
@thang-bit: This is probably caused by mismatched versions of TypeSpec.  Follow the steps here to ensure you are using the latest version:

https://github.com/Azure/azure-rest-api-specs/pull/24937#pullrequestreview-1566171892

Reply:
I'm getting this error when running the final step `npx tsv path/to/my/spec`:
`SyntaxError: The requested module 'node:util' does not provide an export named 'parseArgs'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:128:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:194:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:385:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)`
I'm using node version 16.15.1

Reply:
I believe you need to upgrade to `16.17.0` or higher:

https://nodejs.org/en/blog/release/v16.17.0#notable-changes

Thank you for reporting this, we will update the minimum required node versions in the `tsv` tool.

Reply:
The typespec validation test passed. Thank you, Mike. Can you share with me the tsv tool documentation?

Reply:
The tool does not have documentation yet, but you can view the sources here: https://github.com/Azure/azure-rest-api-specs/tree/main/eng/tools/TypeSpecValidation